### PR TITLE
Added a new dashboard for participating pods in event mesh

### PR DESCRIPTION
**Pull request by sayanh**
**Description**

Changes proposed in this pull request:

- Added a new dashboard for event-mesh pods
- Added label `kyma-project.io/dashboard: "event-mesh"` to facilitate the eventmesh dashboard

**Related issue(s)**

See also https://github.com/kyma-project/kyma/pull/6880

### CODE REVIEW

File: resources/knative-eventing/charts/knative-eventing/templates/eventing.yaml [hunk 1]
```diff
@@ -1623,6 +1623,7 @@ spec:
       labels:
         app: eventing-controller
         eventing.knative.dev/release: "v0.10.0"
+        kyma-project.io/dashboard: "event-mesh"
```
Comment (antoineco) [inline resources/knative-eventing/charts/knative-eventing/templates/eventing.yaml]: (nit) none of those quotes are required in YAML

Comment (sayanh) [reply resources/knative-eventing/charts/knative-eventing/templates/eventing.yaml]: Right, I changed in all the values.

**Pull request merged**